src/dbt_client/dbtCloudIntegration.ts

```diff
@@ -527,6 +533,15 @@ export class DBTCloudProjectIntegration
     );
     command.addArgument("--source");
     command.addArgument("dbt-power-user");
+    const currentVersion = this.getVersion()
+      .map((part) => new String(part))
+      .join(".");
+    const downloadArtifactsVersion = "1.0.1";
+    if (semver.gt(currentVersion, downloadArtifactsVersion)) {
+      if (["parse"].includes(command.args[0])) {
+        command.addArgument("--download-artifacts");
+      }
```

Comment:
@mdesmet I am assuming we are adding download artifact flag to all methods for now and will remove the gradually.

We need to check the methods we use support the flag. Here is a list of method that support the flag as per Ceaser:

dbt build
dbt clean
dbt clone
dbt compile
dbt deps
dbt docs
dbt list
dbt parse
dbt reattach
dbt retry
dbt run
dbt run-operations
dbt seed
dbt show
dbt snapshot
dbt source
dbt test